spec/spec_helper.rb

```diff
@@ -1,12 +1,20 @@
+# frozen_string_literal: true
+
 require 'bundler/setup'
-require 'htmx/rails'
+require 'generator_spec'
+require 'htmx_rails'
+require 'simplecov'
+
+SimpleCov.start do
+  add_filter '/spec/'
+end
 
 RSpec.configure do |config|
   # Enable flags like --only-failures and --next-failure
   config.example_status_persistence_file_path = '.rspec_status'
 
   # Disable RSpec exposing methods globally on `Module` and `main`
-  config.disable_monkey_patching!
+  # config.disable_monkey_patching!
```

Comment:
why is this?

Reply:
I need this for the [stub_const](https://github.com/rootstrap/htmx-rails/blob/install-generators/spec/generators/install_generator_spec.rb#L53) to 'simulate' that Webpack is installed.